src/hipscat_import/catalog/resume_plan.py

```diff
@@ -153,6 +150,9 @@ def read_histogram(self, healpix_order):
         # - combine into a single histogram
         # - write out as a single histogram for future reads
         # - remove all partial histograms
+        remaining_map_files = self.get_remaining_map_keys()
+        if len(remaining_map_files) > 0:
+            raise RuntimeError("some map stages did not complete successfully.")
```

Comment:
Optional comment: This might be more informative as:

```
raise RuntimeError(f"{len(remaining_map_files)} map stages did not complete successfully.")
```

Same below.